src/code/z_actor.c

```diff
@@ -1991,28 +1991,40 @@ void Actor_InitContext(PlayState* play, ActorContext* actorCtx, ActorEntry* play
     func_8002FA60(play);
 }
 
-u32 D_80116068[ACTORCAT_MAX] = {
+u32 sFreezeCategoryPlayerFlags1Masks[ACTORCAT_MAX] = {
```

Comment:
I think `PlayerState` rather than `PlayerFlags`, since firstly the latter is probably going to end up being ambiguous (I'm reasonably sure Player has several other sets of flags apart from the state ones), and secondly it fits better with the enum.